### PR TITLE
Add `flip` icons

### LINKED ISSUE #622: [Icon Requests from Feather] `flip`/`reflect`
As per https://github.com/lucide-icons/lucide/issues/119 and https://github.com/feathericons/feather/issues/426:

image
reflect

### PR DESCRIPTION
**Pull request by ericfennis**
closes #622

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Comparing to other icons it maybe looks a tad too small, but I'm not quite sure.
I also came up with an alternate design last week, when working on some expand/collapse icons:
image
flip

Expand/collapse for reference:
image

File: icons/flip-vertical.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M19 8V6a2 2 0 0 0-2-2H7a2 2 0 0 0-2 2v2" />
+  <path d="M19 16v2a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2v-2" />
```
Comment (karsa-mistmere) [inline icons/flip-vertical.svg:13]:
```suggestion
  <path d="M21 8V5a2 2 0 0 0-2-2H5a2 2 0 0 0-2 2v3" />
  <path d="M21 16v3a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-3" />
```

**Review by karsa-mistmere — COMMENTED** [review]
Not quite sure if this is better, but maybe we could stick with `square`'s dimensions?

**Comment by ericfennis** [comment]
@karsa-mistmere Maybe add the other ones as well?

**Review by karsa-mistmere — COMMENTED** [review]
I made a PR to your fork adding the alternate icons: https://github.com/ericfennis/lucide/pull/3

**Pull request merged**